docs/source/TaskEngineWorkflows.md

```diff
@@ -348,6 +354,39 @@ This workflow allows you to create an MP4 from a VOD asset
 }
 ```
 
+## build_thumbnails
+
+This workflow allows the generation of a thumbnail sprite and vtt file, which can then be used for video timeline preview thumbnails.
```

Comment:
Requires re-wording. Something like:

This workflow allows you to generate thumbnails that can be used for video timeline previews.